### PR TITLE
Functional tests - Change selectors for shop parameters pages

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Change selectors for shop parameters pages
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | `DOWNLOAD_PATH='your_downloaded_path' TEST_PATH="functional/BO/13_shopParameters/*" URL_FO=shopUrl/ npm run specific-test`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18862)

### CODE REVIEW

**Review by SimonGrn — DISMISSED** [review]

File: tests/puppeteer/pages/BO/shopParameters/customerSettings/index.js [hunk 1]
```diff
@@ -11,11 +11,12 @@ module.exports = class customerSettings extends BOBasePage {
...
     this.generalForm = '#configuration_form';
-    this.redisplayCartAtLoginLabel = 'label[for=\'form_general_redisplay_cart_at_login_%TOGGLE\']';
-    this.enablePartnerOfferLabel = 'label[for=\'form_general_enable_offers_%TOGGLE\']';
-    this.sendEmailAfterRegistrationLabel = 'label[for=\'form_general_send_email_after_registration_%TOGGLE\']';
-    this.askForBirthDateLabel = 'label[for=\'form_general_ask_for_birthday_%TOGGLE\']';
-    this.enableB2BModeToggle = 'label[for=\'form_general_enable_b2b_mode_%TOGGLE\']';
+    this.redisplayCartAtLoginLabel = toggle => `label[for='form_general_redisplay_cart_at_login_${toggle}']`;
+    this.enablePartnerOfferLabel = toggle => `label[for='form_general_enable_offers_${toggle}']`;
+    this.sendEmailAfterRegistrationLabel = toggle => `label[for='form_general_send_email_after_registration_${toggle
+    }']`;
```
Comment (Progi1984) [inline tests/puppeteer/pages/BO/shopParameters/customerSettings/index.js]:
```suggestion
    this.sendEmailAfterRegistrationLabel = toggle => `label[for='form_general_send_email_after_registration_${toggle}']`;
```

File: tests/puppeteer/pages/BO/shopParameters/orderSettings/index.js [hunk 2]
```diff
@@ -10,18 +10,23 @@ module.exports = class OrderSettings extends BOBasePage {
 
     // Selectors
     this.generalForm = '#configuration_form';
-    this.enableFinalSummaryLabel = `${this.generalForm} label[for='form_general_enable_final_summary_%TOGGLE']`;
-    this.enableGuestCheckoutLabel = `${this.generalForm} label[for='form_general_enable_guest_checkout_%TOGGLE']`;
-    this.disableReorderingLabel = `${this.generalForm} label[for='form_general_disable_reordering_option_%TOGGLE']`;
+    this.enableFinalSummaryLabel = toggle => `${this.generalForm
+    } label[for='form_general_enable_final_summary_${toggle}']`;
```
Comment (Progi1984) [inline tests/puppeteer/pages/BO/shopParameters/orderSettings/index.js]:
```suggestion
    this.enableFinalSummaryLabel = toggle => `${this.generalForm} label[for='form_general_enable_final_summary_${toggle}']`;
```

File: tests/puppeteer/pages/BO/shopParameters/customerSettings/index.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/puppeteer/pages/BO/shopParameters/customerSettings/index.js]: Eslint error with this suggestion

File: tests/puppeteer/pages/BO/shopParameters/orderSettings/index.js (hunk 2 above)
Comment (boubkerbribri) [reply tests/puppeteer/pages/BO/shopParameters/orderSettings/index.js]: same here

**Review by PierreRambaud — DISMISSED** [review]

File: tests/puppeteer/pages/BO/shopParameters/orderSettings/index.js (hunk 2 above)
Comment (Progi1984) [reply tests/puppeteer/pages/BO/shopParameters/orderSettings/index.js]:
```suggestion
    this.enableFinalSummaryLabel = toggle => `${this.generalForm}`
    + ` label[for='form_general_enable_final_summary_${toggle}']`;
```

File: tests/puppeteer/pages/BO/shopParameters/customerSettings/index.js (hunk 1 above)
Comment (Progi1984) [reply tests/puppeteer/pages/BO/shopParameters/customerSettings/index.js]:
```suggestion
    this.sendEmailAfterRegistrationLabel = toggle => `label`
    + `[for='form_general_send_email_after_registration_${toggle}']`;
```
for all long lines

Comment (boubkerbribri) [reply tests/puppeteer/pages/BO/shopParameters/customerSettings/index.js]: done

**Review by NeOMakinG — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Pull request merged**